.github/workflows/ci-disable-gtest.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
       with:
         path: ext
-        key: ${{ runner.os }}-disable-gtest-${{ hashFiles('ext/*.cmd') }}
+        key: ${{ runner.os }}-disable-gtest-${{ hashFiles('CMakeLists.txt', 'ext/*.cmd') }}
```

Comment:
Frankie: The dependency of the `ext` cache files on CMakeLists.txt seems wrong. The contents of the `ext` directory do not depend on CMakeLists.txt. Could you explain why this is necessary?

Reply:
Oh... so, it looks like I had encountered issues because the cache action is here, but then the `sed` that was replacing the cmake flags happens below. So when I removed the sed action it was still using the cached ext folder with the shared libraries. So perhaps the appropriate fix would be to modify the cache key slightly, without having it depend on the contents of CMakeLists.txt? (which as you correctly point out doesn't affect the builds in the ext directory).

Reply:
Actually, since this pull request will be merged in all at once, and since it involves changes to the ext/*.cmd files, I'll just undo these changes.